ar4_gazebo_sim/LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
What advantage do we have in having `ar4_gazebo_sim` and `ar4_gazebo_bringup` as separate packages instead of just one `ar4_gazebo`?

Reply:
Arguably, it would be totally possible have just one, so you can say this is mostly personal preference.

However, out of experience I find that keeping them separate allows to have a launch file that will only launch the simulation (or hardware) parts without the common parts, which comes handy when debugging the hardware/simulation layer.

Bringup packages end up being only a front-end for the packages needed for any particular configuration, and in the case of simulation, it's the single fail-safe place to set the use_sim_time param for the whole set without risking omissions.

Regarding naming, packages starting a robot (or a simulation of one) [must end in "bringup"](https://ros.org/reps/rep-0144.html#special-cases) to give a hint to the user of their function.